### ISSUE TITLE
package.json main

### ISSUE THREAD

**Issue by pqvst** [JonDum/ractive-datatable#8]
Wouldn't it make more sense if `"main"` in `package.json` pointed to `"datatable.min.js"`? It seems weird that it points to the src js file. As a user I want to use the built version - not the source file.

That way I could simply include ractive-datatable in browserify:

```
browserify([
    "ractive",
    "ractive-datatable"
]);
```

I guess for now I have to do something like:

```
browserify([
    "ractive",
    "./node_modules/ractive-datatable/datatable.min.js"
]);
```

Or?

**Comment by sarendipitee** [comment]
For me and my team's use case it's more efficient to point to the src, since it doesn't contain extra UMD loaders and webpack will pick it up like a first class module. Before, I didn't even include a built version. I include it as an after thought mainly for people with no build systems at all. 

I'm not that familiar with Browserify, though. Why can't it read the package.main and use the version in src?

**Comment by paulocoghi** [comment]
I agree with @JonDum . When you don't use a build system, use directly `datatable.min.js`.

I am using rollup to bundle some packages with my own, and the ones that point `main` to a build file spoils things a bit, because the extra UMD loaders.
